libcontainer/console.go

```diff
@@ -17,57 +15,3 @@ type Console interface {
 	// Fd returns the fd for the master of the pty.
 	File() *os.File
 }
-
-const (
-	TerminalInfoVersion uint32 = 201610041
-	TerminalInfoType    uint8  = 'T'
-)
-
-// TerminalInfo is the structure which is passed as the non-ancillary data
-// in the sendmsg(2) call when runc is run with --console-socket. It
-// contains some information about the container which the console master fd
-// relates to (to allow for consumers to use a single unix socket to handle
-// multiple containers). This structure will probably move to runtime-spec
-// at some point. But for now it lies in libcontainer.
-type TerminalInfo struct {
```

Comment:
I think we can create a new socket for every container for now, at least before we figure out the double socket issue.